recipes/hug/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "hug" %}
+{% set version = "2.1.2" %}
+{% set sha256 = "a9bd00e14549001d89b4cff08742d9da50f2f6a1fb0cf9c872bfb405e66d0a51" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/timothycrosley/hug/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [py2k]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - hug = hug:development_runner.hug.interface.cli
+
+requirements:
```

Comment:
Yes, sorry, that is what I meant.

Normally we try to keep this section lean to avoid potential issues that we have seen with `conda-build` compiling Python files it shouldn't and to avoid potential issues with `conda-build-all`/`conda-smithy` due to dependency resolution. The way these problems manifest can be subtle and surprising so it is easiest to just eliminate potential causes.